### PR TITLE
Update coffee rails

### LINKED ISSUE #3481: Update Coffee-rails?
Responding to ` GitHub found 1 vulnerability on anitagraham/gemname's default branch (1 low).` 
Following up I found:

```
Upgrade actionpack to version 6.0.3.2 or later. 
...
CVE-2020-8185
low severity
Vulnerable versions: >= 6.0.0, <= 6.0.3.1
Patched version: 6.0.3.2 
```

A few `bundle update`s later I came to 

```
  refinerycms was resolved to 4.1.0, which depends on
      refinerycms-core (= 4.1.0) was resolved to 4.1.0, which depends on
        coffee-rails (>= 4.0.0, ~> 4.0)
```

Is there any reason we cannot update to a later version of coffee-rails?

thanks

Anita

### PR DESCRIPTION
**Pull request by anitagraham**
Update to latest version of coffee-rails. Required for using Rails 6.0.3.2 (see #3481)

This required an update to refinerycms-wymeditor version 3.0.0

Closes #3481

### CODE REVIEW

**Comment by jbanahan** [comment]
Can I ask why you're removing the contents of the readme?

**Comment by parndt** [comment]
> Can I ask why you're removing the contents of the readme?

fixed that :smile:

**Comment by parndt** [comment]
> Note: I needed to use the master version of refinerycms-wymeditor to get the tests passing. I'll add an issue there to get a new release, if that's the right process.

A release is coming

**Comment by parndt** [comment]
> A release is coming

[refinerycms-wymeditor version 3.0.0 released](https://rubygems.org/gems/refinerycms-wymeditor/versions/3.0.0)

File: Gemfile [hunk 1]
```diff
@@ -17,7 +17,8 @@ gem 'routing-filter', github: 'svenfuchs/routing-filter', branch: 'master'
 gem 'refinerycms-acts-as-indexed', ['~> 3.0', '>= 3.0.0']
 
 # Add the default visual editor, for now.
-gem 'refinerycms-wymeditor', ['~> 2.2', '>= 2.2.0']
+# gem 'refinerycms-wymeditor', ['~> 2.2', '>= 2.2.0']
+gem 'refinerycms-wymeditor', git: 'https://github.com/parndt/refinerycms-wymeditor', branch: 'master'
```
Comment (parndt) [inline Gemfile:21]:
```suggestion
gem 'refinerycms-wymeditor', ['~> 3.0', '>= 3.0.0']
```

**Review by parndt — APPROVED** [review]

**Pull request merged**